target/linux/ath79/dts/ar9341_engenius_ens202ext.dts

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9341.dtsi"
+
+/ {
+	model = "Engenius ENS202EXT";
+	compatible = "engenius,ens202ext", "qca,ar9341";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "Reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		system: power {
```

Comment:
`led_power: power {`

Reply:
Then I should change all the aliases at the top from "&system" to "%led_power"?

Reply:
Does this include removing `debounce-interval = <60>;`?

Reply:
aliases: yes
debounce: there are different opinions on this. Never had the chance to really check it.